### PR TITLE
Update using-multiple-gatsby-themes.md

### PR DESCRIPTION
**Pull request by JanLawrence**
Update file for translation to Spanish in 'docs/docs/themes'

### CODE REVIEW

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ module.exports = {
...
 
 ![The homepage of the site created by gatsby-theme-starter](../images/gatsby-theme-starter-home.png)
 
 ![The `notes` route of a site created by gatsby-theme starter](../images/gatsby-theme-starter-notes.png)
 
-## Tutorial
+## El tutorial
 
-For a step-by-step tutorial, see the ["Using Multiple Themes Together" tutorial](/tutorial/using-multiple-themes-together).
+Para ver un tutorial paso a paso, consulte el tutorial ["Uso de varios temas juntos"](/tutorial/using-multiple-themes-together).
```
Comment (kbravh) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
Para ver un tutorial paso a paso, consulta el tutorial ["Usar varios temas juntos"](/tutorial/using-multiple-themes-together).
```

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ module.exports = {
 }
 ```
 
-In the default setup, a blog will be served from the root path (`/`), and the notes content will be served from `/notes`.
+En la configuración predeterminada, se servirá un blog desde la ruta raíz (`/`), y el contenido de las notas se servirá desde `/notes`.
 
-Run `gatsby develop` to start a development server and view your the site:
+Ejecute `gatsby develop` para iniciar un servidor de desarrollo y ver si es el sitio:
```
Comment (kbravh) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
Ejecuta `gatsby develop` para iniciar un servidor de desarrollo y ver el sitio:
```

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 3]
```diff
@@ -32,14 +32,14 @@ module.exports = {
 }
 ```
 
-In the default setup, a blog will be served from the root path (`/`), and the notes content will be served from `/notes`.
+En la configuración predeterminada, se servirá un blog desde la ruta raíz (`/`), y el contenido de las notas se servirá desde `/notes`.
```
Comment (kbravh) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
Con la configuración predeterminada, se servirá un blog desde la ruta root (`/`), y el contenido de las notas se servirá desde `/notes`.
```

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
-The `gatsby-starter-theme` composes two Gatsby themes: `gatsby-theme-blog` and `gatsby-theme-notes`
+La `gatsby-starter-theme` compone dos temas de Gatsby: `gatsby-theme-blog` y `gatsby-theme-notes`
 
 ```shell
 gatsby new my-notes-blog https://github.com/gatsbyjs/gatsby-starter-theme
 ```
 
-The starter includes both theme packages (`gatsby-theme-blog` and `gatsby-theme-notes`) in the starter's `gatsby-config.js` file.
+El inicio incluye ambos paquetes de tema (`gatsby-theme-blog` y `gatsby-theme-notes`) en el archivo `gatsby-config.js` del starter.
```
Comment (kbravh) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
El starter incluye ambos paquetes de tema (`gatsby-theme-blog` y `gatsby-theme-notes`) en el archivo `gatsby-config.js` del starter.
```

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 5]
```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Multiple Gatsby Themes
+título: Usando de múltiples temas de Gatsby
 ---
 
-Gatsby themes are intended to be composable. This means you can install multiple themes alongside each other.
+Gatsby temas están destinados a ser componible. Esto significa que puede instalar temas múltiples el uno junto al otro.
 
-The `gatsby-starter-theme` composes two Gatsby themes: `gatsby-theme-blog` and `gatsby-theme-notes`
+La `gatsby-starter-theme` compone dos temas de Gatsby: `gatsby-theme-blog` y `gatsby-theme-notes`
```
Comment (kbravh) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
La `gatsby-starter-theme` se compone de dos temas de Gatsby: `gatsby-theme-blog` y `gatsby-theme-notes`
```

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Multiple Gatsby Themes
+título: Usando de múltiples temas de Gatsby
 ---
 
-Gatsby themes are intended to be composable. This means you can install multiple themes alongside each other.
+Gatsby temas están destinados a ser componible. Esto significa que puede instalar temas múltiples el uno junto al otro.
```
Comment (kbravh) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
Gatsby temas están destinados a ser combinados. Esto significa que puedes instalar varios temas el uno junto al otro.
```

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 7]
```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Multiple Gatsby Themes
+título: Usando de múltiples temas de Gatsby
```
Comment (kbravh) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
título: Usar múltiples temas de Gatsby
```

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 6 above)
Comment (nucliweb) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Gatsby temas están destinados a ser combinados. => Los temas Gatsby están destinados a ser combinados.

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 5 above)
Comment (nucliweb) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: La `gatsby-starter-theme` => El `gatsby-starter-theme`.

Tema en castellano es masculino.

**e.g.**
[Los temas de sitios que no sean GPL...](https://es.wordpress.org/themes/getting-started/)

**Comment by Darking360** [comment]
@JanLawrence Recuerda agregar los comentarios para poder aporbar este PR :rocket:

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 1 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 2 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 4 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 6 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 7 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 5 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 3 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 8]
```diff
@@ -1,16 +1,16 @@
 ---
-title: Using Multiple Gatsby Themes
+título: Usar múltiples temas de Gatsby
```
Comment (xmartinezpujol) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
title: Usar múltiples temas de Gatsby
```

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 9]
```diff
@@ -32,14 +32,14 @@ module.exports = {
...
 
-Run `gatsby develop` to start a development server and view your the site:
+Ejecuta `gatsby develop` para iniciar un servidor de desarrollo y ver el sitio:
 
 ![The homepage of the site created by gatsby-theme-starter](../images/gatsby-theme-starter-home.png)
 
 ![The `notes` route of a site created by gatsby-theme starter](../images/gatsby-theme-starter-notes.png)
 
-## Tutorial
+## El tutorial
```
Comment (xmartinezpujol) [inline docs/docs/themes/using-multiple-gatsby-themes.md]:
```suggestion
## Tutorial
```

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 8 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md (hunk 9 above)
Comment (JanLawrence) [reply docs/docs/themes/using-multiple-gatsby-themes.md]: Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.

File: docs/docs/themes/using-multiple-gatsby-themes.md [hunk 10]
```diff
@@ -40,6 +40,6 @@ Ejecuta `gatsby develop` para iniciar un servidor de desarrollo y ver el sitio:
 
 ![The `notes` route of a site created by gatsby-theme starter](../images/gatsby-theme-starter-notes.png)
```
Comment (gusma) [inline docs/docs/themes/using-multiple-gatsby-themes.md]: @JanLawrence @horacioh @Darking360 ¿Debiéramos dejar esto igual?

**Pull request merged**